distribution/zip/ballerina/src/assembly/bin.xml

```diff
@@ -165,6 +165,12 @@
             </directory>
             <outputDirectory>src</outputDirectory>
         </fileSet>
+        <fileSet>
+            <directory>
```

Comment:
I think this won't work anymore as source files are not there inside the jar anymore